### PR TITLE
Fixed certain unicode characters in users' paths causing issues on Windows

### PR DESCRIPTION
**Pull request by friedkeenan**
## Changelog / Overview

What will *probably* be the change is that in `SceneFileWriter.combine_movie_files`, I'll add `encoding="utf-8"` to the `open` call, but first I want to ensure that the test I added fails on Windows.

## Motivation

A user on the discord had rendering fail for them because they had Chinese characters in their folder name. Users should be able to use whatever language they want in their folder names.

## Explanation for Changes

The test uses the afflicting characters as the name of its media directory, which until I fix the issue should cause it to fail when writing the partial movie paths for ffmpeg. It fails because Windows' default file encoding isn't UTF-8 but rather some zany thing.

## Testing Status

Tests pass on my local linux installation but that's to be expected.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new functions/classes either have a docstring or are private
- [x] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by friedkeenan** [comment]
Tests failing correctly, time to commit the fix

**Comment by friedkeenan** [comment]
Ok, turns out it's failing when outputting logs which include the media directory, which is a related but separate issue than I was trying to fix. I guess let me look into that

**Comment by Darylgolden** [comment]
Hi, are you still working on the PR? Due to refactoring plans, if this PR is not ready to be merged by the end of the year, it may have to be closed.

File: tests/test_scene_rendering/test_file_writer.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import sys
+
+import pytest
+
+from ..utils.commands import capture
```
Comment (behackl) [inline tests/test_scene_rendering/test_file_writer.py:5]:
```suggestion
from manim.utils.commands import capture
```

**Review by behackl — APPROVED** [review]
I am not sure what exactly it was that we did, but it seems this issue has been resolved in the meantime. After fixing the import, the tests pass. 🤷

**Comment by Darylgolden** [comment]
On Windows, can confirm this PR fixed the issue.

**Pull request merged**